packages/editor/src/components/properties/InstancingMeshProperties.tsx

```diff
@@ -50,16 +50,15 @@ export default function InstancingMeshProperties({
 }) {
   const value = state.value
   const props = value as MeshProperties
-
   const { t } = useTranslation()
 
   const initialMeshes = traverseScene(
     (node) => {
-      const group = getComponent(node, GroupComponent)
+      const group = useComponent(node, GroupComponent)
```

Comment:
this is not correct, hooks should only be used in the body of a react function

Reply:
all right